basic/src/tasks/components/TaskList.tsx

```diff
@@ -0,0 +1,53 @@
+import { deleteTasks, getTasks, useQuery } from "wasp/client/operations";
+import { TaskListItem } from "./TaskListItem";
+
+export function TaskList() {
+  const { data: tasks, isLoading, isSuccess } = useQuery(getTasks);
+
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
+  if (!isSuccess) {
+    return <p>Error loading tasks.</p>;
+  }
+
+  const completedTasks = tasks.filter((task) => task.isDone);
+
+  return (
+    <div className="flex flex-col gap-6">
+      <div className="flex items-center justify-between">
+        <div>
+          <span>
+            {tasks.length} {tasks.length === 1 ? "task" : "tasks"}
+          </span>
+          <span className="mx-2">·</span>
+          <span>{completedTasks.length} completed</span>
+        </div>
+        {completedTasks.length > 0 && (
+          <button className="font-medium" onClick={deleteCompletedTasks}>
+            🗑️ Clear completed
```

Comment:
```suggestion
            Clear completed
```

Personal preference to drop the emojis to keep the app looking cleaner.

Reply:
I'm was too lazy to find svgs.

Reply:
Replaced with svg.